tests/python/frontend/tflite/test_forward.py

```diff
@@ -2618,6 +2617,27 @@ def test_forward_select():
             )
 
 
+def _test_fake_quant(value, min, max, num_bits):
```

Comment:
Rather than using this additional function, can you use [pytest.mark.parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html) ?

Something like:
```
@pytest.mark.parametrize("quant_bits", [2, 4, 8, 16])
@pytest.mark.parametrize("value,min,max", [
  [-10.11, -6, 6],
  // .. more here ...
])
def test_fake_quant(value, min, max, quant_bits):
```

Reply:
Thank you for your great suggestion. Updated according to your suggestion.